go-controller/pkg/ovn/multihoming_test.go

```diff
@@ -502,6 +503,22 @@ func newMultiHomedPod(testPod testPod, multiHomingConfigs ...userDefinedNetInfo)
 	return pod
 }
 
+func generateDefaultNSEAnnotation(multiHomingConf userDefinedNetInfo) string {
+	nadNamePair := strings.Split(multiHomingConf.nadName, "/")
+	if len(nadNamePair) != 2 {
```

Comment:
we can't have the format of omitting the namespace, right ? Since the NAD must be in the ovn-k ns, right ?

Reply:
OK, let's move on then. Tks.